pyiceberg/io/pyarrow.py

```diff
@@ -1237,16 +1238,29 @@ def _task_to_record_batches(
         # When V3 support is introduced, we will update `downcast_ns_timestamp_to_us` flag based on
         # the table format version.
         file_schema = pyarrow_to_schema(physical_schema, name_mapping, downcast_ns_timestamp_to_us=True)
+
+        if file_schema is None:
```

Comment:
is there a case where `file_schema` is None?

Reply:
I am not sure, but having a null `file_schema` wouldn't make sense.